blank_rust_project/src/main.test.js

```diff
@@ -0,0 +1,50 @@
+const nearlib = require('nearlib');
+const CONTRACT_NAME = 'expl3-test';
+
+function getConfig(env) {
+    switch (env) {
+
+    case 'production':
+    case 'development':
```

Comment:
shouldn't test use `ci` environment? Any reason to use special config for test?

Reply:
This is how the other projects are set up too at the moment. I am in favor of changing it, but would slightly prefer to do it everywhere at once in a follow up PR.

Reply:
>  I am in favor of changing it, but would slightly prefer to do it everywhere at once in a follow up PR.

sure. Dunno how it ended up like this, I thought we had custom test_environment.js instead